_stbt/utils.py

```diff
@@ -80,7 +80,7 @@ def find_import_name(filename):
 
 def to_bytes(text):
     if isinstance(text, str):
-        return text.encode("utf-8")
+        return text.encode("utf-8", errors="backslashreplace")
```

Comment:
I don't think this can fail?  All unicode sequences should be representable as utf-8 unless I'm missing something?

Reply:
I'm not sure. What's the point of having `errors` at all on `encode()`?

Reply:
For encodings where the whole of unicode can't be represented.  e.g. ASCII

```
$ ipython3
In [2]: 'Röthlis'.encode('ASCII')
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-2-10281243efbf> in <module>()
----> 1 'Röthlis'.encode('ASCII')

UnicodeEncodeError: 'ascii' codec can't encode character '\xf6' in position 1: ordinal not in range(128)

In [3]: 'Röthlis'.encode('ASCII', errors="backslashreplace")
Out[3]: b'R\\xf6thlis'
```